mkdocs.yml

```diff
@@ -46,40 +68,137 @@ nav:
       - min: asl/ref/min.md
       - sum: asl/ref/sum.md
     - Math:
+      - abs: asl/ref/abs.md
+      - add: asl/ref/add.md
       - and: asl/ref/and.md
+      - avg: asl/ref/avg.md
       - bottomk: asl/ref/bottomk.md
       - bottomk-others-avg: asl/ref/bottomk-others-avg.md
       - bottomk-others-max: asl/ref/bottomk-others-max.md
       - bottomk-others-min: asl/ref/bottomk-others-min.md
       - bottomk-others-sum: asl/ref/bottomk-others-sum.md
       - by: asl/ref/by.md
+      - clamp-max: asl/ref/clamp-max.md
+      - clamp-min: asl/ref/clamp-min.md
       - count: asl/ref/count.md
+      - cq: asl/ref/cq.md
+      - dist-avg: asl/ref/dist-avg.md
+      - dist-max: asl/ref/dist-max.md
+      - dist-stddev: asl/ref/dist-stddev.md
+      - div: asl/ref/div.md
+      - fadd: asl/ref/fadd.md
+      - fdiv: asl/ref/fdiv.md
       - filter: asl/ref/filter.md
+      - fmul: asl/ref/fmul.md
+      - fsub: asl/ref/fsub.md
       - ge: asl/ref/ge.md
       - gt: asl/ref/gt.md
       - le: asl/ref/le.md
       - lt: asl/ref/lt.md
       - max: asl/ref/max.md
+      - median: asl/ref/median.md
+      - mul: asl/ref/mul.md
       - min: asl/ref/min.md
+      - neg: asl/ref/neg.md
       - or: asl/ref/or.md
+      - pct: asl/ref/pct.md
+      - per-step: asl/ref/per-step.md
+      - percentiles: asl/ref/percentiles.md
+      - pow: asl/ref/pow.md
+      - random: asl/ref/random.md
+      - sqrt: asl/ref/sqrt.md
+      - srandom: asl/ref/srandom.md
       - stat: asl/ref/stat.md
       - stat-avg: asl/ref/stat-avg.md
       - stat-count: asl/ref/stat-count.md
       - stat-last: asl/ref/stat-last.md
       - stat-max: asl/ref/stat-max.md
       - stat-min: asl/ref/stat-min.md
       - stat-total: asl/ref/stat-total.md
+      - stddev: asl/ref/stddev.md
+      - sub: asl/ref/sub.md
+      - sum: asl/ref/sum.md
+      - time: asl/ref/time.md
+      - time-span: asl/ref/time-span.md
       - topk: asl/ref/topk.md
       - topk-others-avg: asl/ref/topk-others-avg.md
       - topk-others-max: asl/ref/topk-others-max.md
       - topk-others-min: asl/ref/topk-others-min.md
       - topk-others-sum: asl/ref/topk-others-sum.md
-      - sum: asl/ref/sum.md
+    - Stack Manipulation:
```

Comment:
Can we move this after presentation? That way the high level categories follow the general expression order.